docs/public/demos/theme.css

```
@@ -0,0 +1,254 @@
+  /* Taskito brand green (matches the product/dashboard). Var names kept for compatibility. */
+  --indigo:#1f9d54;
+  --indigo-br:#3bbf72;
+  --indigo-dk:#15803d;
+  --indigo-soft:rgba(31,157,84,.12);
+  --indigo-line:rgba(31,157,84,.26);
+  --cyan:#29b8a8;
+  --grn:#4ec985;
+  --amber:#ffb86b;
+  --red:#ff6b6b;
+
+  --mono:'IBM Plex Mono',ui-monospace,SFMono-Regular,monospace;
```

Stylelint (17.13.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)